SoftLayer/CLI/virt/billing.py

```diff
@@ -28,11 +28,11 @@ def cli(env, identifier):
     table.add_row(['Billing Item Id', utils.lookup(result, 'billingItem', 'id')])
     table.add_row(['Recurring Fee', utils.lookup(result, 'billingItem', 'recurringFee')])
     table.add_row(['Total', utils.lookup(result, 'billingItem', 'nextInvoiceTotalRecurringAmount')])
-    table.add_row(['Provision Date', utils.lookup(result, 'billingItem', 'provisionDate')])
+    table.add_row(['Provision Date', utils.lookup(result, 'provisionDate')])
 
-    price_table = formatting.Table(['Recurring Price'])
+    price_table = formatting.Table(['description', 'Recurring Price'])
```

Comment:
Table headings should start with a capital letter.

```suggestion
    price_table = formatting.Table(['Description', 'Recurring Price'])
```